sdk/tests/00_test_list.txt

```diff
@@ -3,5 +3,7 @@
 
 conformance/00_test_list.txt
 conformance/more/00_test_list.txt
+deqp/data/gles2/shaders/00_test_list.txt
+deqp/data/gles3/shaders/00_test_list.txt
```

Comment:
Add "--min-version 2.0.0" at the beginning of this line, which will restrict the GLES3 shaders to only run as part of the WebGL 2.0 conformance suite.